lucida/commandcenter/static/js/speech-streamer.js

```diff
@@ -0,0 +1,69 @@
+var transcript = '';
+var recognizing = false;
+var micLoc = 'static/image/';
+if (!('webkitSpeechRecognition' in window)) {
```

Comment:
does a warning show up if you're not using chrome or what if a poor soul for some reason is using Firefox?